chain/chunks/src/shards_manager_actor.rs

```diff
@@ -1553,13 +1559,13 @@ impl ShardsManagerActor {
                 partial_encoded_chunk.header.shard_id(),
             ) {
                 if hash != chunk_hash {
-                    warn!(
+                    tracing::warn!(
                         target: "client",
-                        "Rejecting un-requested chunk {:?}, height {}, shard_id {}, because of having {:?}",
-                        chunk_hash,
-                        partial_encoded_chunk.header.height_created(),
-                        partial_encoded_chunk.header.shard_id(),
-                        hash
+                        ?chunk_hash,
+                        height = %partial_encoded_chunk.header.height_created(),
+                        shard_id = %partial_encoded_chunk.header.shard_id(),
+                        ?hash,
+                        "rejecting un-requested chunk because of having"
```

Comment:
this message needs to be updated as well, `because of having` makes little sense